sources/packages/backend/libs/services/src/constants/system-configurations-constants.ts

```diff
@@ -31,3 +31,5 @@ export const QUEUE_RETRY_DEFAULT_CONFIG = {
    */
   backoff: QUEUE_RETRY_DELAY,
 };
+
+export const IER_SCHEDULER_JOB_ID = "IER_SCHEDULER_JOB_ID";
```

Comment:
Maybe we can use the queue name here also. The one defined in the enum `QueueNames`.